src/subclass/types.rs

```diff
@@ -16,6 +16,7 @@ use translate::*;
 use wrapper::Wrapper;
 use {IsA, IsClassFor, Object, StaticType, Type};
 
+use super::super::object::ObjectExt;
```

Comment:
That's the first time I see `super::super`. Just asking but I guess the answer is no: is it possible to not import it this way? It seems very werid?

Reply:
You could also do `use ::object::ObjectExt`. Do you prefer that?

Reply:
Changed that

Reply:
New version pushed that also changes the name of the function (to be shorter) and makes it more generic

Reply:
All better then! Thanks for doing the change. :)